modules/gke/variables.tf

```diff
@@ -11,4 +11,23 @@ variable "location"{
 variable "initial_node_count"{
     description = "number of nodes in the default node pool"
     default = 1
+}
+
+variable "remove_default_node_pool"{
+    description = "Are you using a separate node pool"
+    default = false
```

Comment:
I am wondering, since we updated the module to now create a node pool, should this be defaulted to true?  otherwise you would have to override it from the gke.tf configuraiton.

Reply:
My reasoning was if we ever want to use the default node pool, this option is baked in, but I'm assuming we want it to default to using a separate node pool moving forward. I'll edit this so what its "true" as the default.